package.json

```
@@ -1418,14 +1433,26 @@
 					"command": "issue.copyGithubPermalink",
 					"when": "github:hasGitHubRemotes"
 				},
+				{
+					"command": "issue.copyGithubPermalinkFile",
+					"when": "false"
+				},
 				{
 					"command": "issue.copyGithubHeadLink",
 					"when": "github:hasGitHubRemotes"
 				},
+				{
+					"command": "issue.copyGithubHeadLinkFile",
+					"when": "github:hasGitHubRemotes"
```

This `when` should also be be false

```suggestion
					"when": "false"
```